mklive.sh

```diff
@@ -510,11 +510,11 @@ while getopts "a:b:r:c:C:T:Kk:l:i:I:S:e:s:o:p:g:v:P:Vh" opt; do
 		l) LOCALE="$OPTARG";;
 		i) INITRAMFS_COMPRESSION="$OPTARG";;
 		I) INCLUDE_DIRS+=("$OPTARG");;
-		S) SERVICE_LIST="$SERVICE_LIST $OPTARG";;
+		S) SERVICE_LIST+=" $OPTARG";;
```

Comment:
this isn't necessarily wrong, but it was valid before. if anything, `SERVICE_LIST` should be made into an array too.